src/scripts/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+const thumbs = document.querySelector('#thumbs');
+const image = document.querySelector('#largeImg');
+
+thumbs.addEventListener('click', (ev) => {
+  ev.preventDefault();
```

Comment:
The `ev.preventDefault()` is applied to all clicks within the `thumbs` element. Consider moving this inside the condition where both `link` and `thumb` are valid to prevent default behavior only when necessary.